docs/clients/README.md

```diff
@@ -16,3 +16,68 @@ Light-clients enable users to interact with your application without having to d
 
 - [Command-Line interface for SDK-based blockchain](./cli.md)
 - [Service provider doc](./service-providers.md)
+
+## Genesis upgrade
+
+If you need to upgrade your node you could export the genesis and migrate it to the new version through this script:
+
+```bash
+go run contrib/export/main.go genesis v0.36 genesis_0_34.json [--time "2019-04-22T17:00:11Z"] [--chain-id test] > ~/.gaiad/genesis.json 
+```
+
+Example from old version:
```

Comment:
I believe the examples/code below should be removed. We'll have a specific Wiki for this and the general client README shouldn't be polluted 👍

Reply:
I just move those to the Wiki, sounds good? It's a copy-pastable example with 2 migrations and I liked to include the fact that `gaia` was in the SDK and now it's a separate repo as I've seen some people having trouble on few issues